python/pyspark/sql/tests/test_pandas_udf_typehints_with_future_annotations.py

```diff
@@ -19,7 +19,7 @@
 import sys
 import unittest
 from inspect import signature
-from typing import Union, Iterator, Tuple, cast, get_type_hints
+from typing import Any, Union, Iterator, Tuple, cast, get_type_hints
```

Comment:
unnecessary change?

Reply:
Oh, btw don't we check the unused import by linter ??

Reply:
We used to, but it seems like excluding F403 in 90003398745bfee78416074ed786e986fcb2c8c\d messed up that. I am looking into this.